styles/styles.css

```diff
@@ -0,0 +1,60 @@
+@import url('styles/fonts.css');
+
+.advantages {
+    width: 1440px;
+    margin: 0 auto;
+    padding: 120px 16px 120px;
+}
+
+.title {
+    text-align: center;
+    font-family: 'Roboto medium', sans-serif;
+    font-size: 40px;
+}
+
+.line {
+    max-width: 1440px;
+    width: 100px;
+    height: 4px;
+    background: #f5bd41;
+    margin: 20px auto 85px;
+}
+
+.row {
+    display: flex;
+    gap: 16px;
+    justify-content: center;
+}
+
+.col {
+    width: 300px;
+    height: 180px;
+    text-align: center;
+    font-family: 'Roboto regular', sans-serif;
+    font-size: 24px;
+}
+
+.footer {
+    width: 1440px;
```

Comment:
You need to transfer property max-width in class .menu-row and delete here
It is allow to cover all space in footer